### ISSUE TITLE
Some titles from Steam library are missing

### ISSUE THREAD

**Issue by dandepeched** [FriendsOfGalaxy/galaxy-integration-steam#23]
Hello, I have issue similar to #18 , but it looks like my public profile is showing those missing games.
There are lots of missing titles, one of the example is all 3 games from "Max Payne" series.

My public profile - https://steamcommunity.com/id/dandepeched/games/?tab=all

I've tried to re-download and reconnect plugin but with no success.
Logs - [steam-plugin-logs.zip](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/files/3921463/steam-plugin-logs.zip)

**Comment by FriendsOfGalaxy** [comment]
Hey Dan,
that's strange - I couldn't find out what is going wrong.

A new release with fixed logging was pushed recently. Could you please recheck and attach logs again?

**Comment by dandepeched** [comment]
Here are new logs - 
[logs.zip](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/files/3951809/logs.zip)

I also checked id for Max Payne (12140) and it is present in 'game_ids' in log.

**Comment by FriendsOfGalaxy** [comment]
Hey, so this is Galaxy issue. Please report using Mantis. They will probably need fresh logs, so refresh integrations before sending report

**Comment by dandepeched** [comment]
> 
> 
> Hey, so this is Galaxy issue. Please report using Mantis. They will probably need fresh logs, so refresh integrations before sending report

Thanks, I will report it. Have you had a chance to look into new logs? Maybe there is some error in it too?

**Comment by dandepeched** [comment]
Issue was reported - https://mantis2.gog.com/view.php?id=27503
Also someone reported similar issue few minutes before me :) https://mantis2.gog.com/view.php?id=27500

**Comment by b1oki** [comment]
Not sure if this is a similar problem, but write here. Two different titles are displayed as one game.
- Gothic
- Gothic Playable Teaser
![image](https://user-images.githubusercontent.com/1561897/79072054-61a29500-7cf8-11ea-95d8-417f819b7de0.png)

**Comment by FriendsOfGalaxy** [comment]
@b1oki yes it is Galaxy issue as well but different one. Remedy remains the same: if you think this should be un-grouped create ticket in Galaxy

**Comment by th3math3us** [comment]
I have some games missing in my library

Company of Heroes
Company of Heroes - Legacy Edition
Company of Heroes: Eastern Front
Company of Heroes: Europe at War
Company of Heroes: Far East War
Company of Heroes: The Great War 1918
Company of Heroes: Modern Combat
Earth 2150 Trilogy
Entropy : Zero
Wilson Chronicles
Half-Life: C.A.G.E.D.
Rexaura
The Deed
Pinball FX3
Polyball
PAC-MAN™ CHAMPIONSHIP EDITION 2
Showdown Bandit
The LEGO® NINJAGO® Movie Video Game
10 Second Ninja X
Mafia II (Classic)
Mafia II: Definitive Edition
Mafia III: Definitive Edition

I don't know if you guys can do something about it

I reported some of them on mantis2 some months ago, but nothing changed :(

**Comment by jdetraz** [comment]
Noticed a similar issue with Dota 2. It seems the plugin is reading some data (time played) but does not import the game. ID is 570.

[logs.zip](https://github.com/FriendsOfGalaxy/galaxy-integration-steam/files/4928585/logs.zip)

**Comment by RelativeProgramming** [comment]
I also have "2064: Read Only Memories" missing from my library. I redeemed that game from Humble Bundle as a Steam Key. However other games, that I acquired the same way, show up without any problems.

**Comment by redeyedfigure** [comment]
the game vampyr is not showing up in my library

**Comment by FriendsOfGalaxy** [comment]
@th3math3us @redeyedfigure @RelativeProgramming follow #76 to verify if that Galaxy problem or not

**Comment by Artyom1Artyom** [comment]
In my Steam account there is a game Eath 2150 Trilogy, this game after purchase was distributed in the list of available games in the following sequence: Eath 2150 Trilogy, Moon Project, lost souls. In gog galaxy  "Eath 2150 Trilogy" is not displayed, Moon Project, lost souls are displayed. On Steam, Eath 2150 Trilogy - aka Earth 2150: Escape from the Blue Planet, but for some reason it's called Trilogy. Can this be fixed to make the Eath 2150 Trilogy appear in gog galaxy?

**Comment by FriendsOfGalaxy** [comment]
Guys, could you check if missing games started to appear when using `public_profiles` mode?
You can configure it with https://github.com/FriendsOfGalaxy/galaxy-integration-steam/blob/master/README.md#configuration-of-the-backend-operation-mode
In any problems reconnect integration after changing the config

**Comment by FriendsOfGalaxy** [comment]
@Artyom1Artyom on the first glance looks like Galaxy isseu. Please follow the instruction here - https://github.com/FriendsOfGalaxy/galaxy-integration-steam/wiki/Common-problems#some-games-are-missing and create separate issue if it plugin problem for this particular game or report via Galaxy "Report issue" option.
